kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/ErrantRecordHandler.java

```diff
@@ -0,0 +1,48 @@
+package com.wepay.kafka.connect.bigquery;
+
+import org.apache.kafka.connect.sink.ErrantRecordReporter;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+public class ErrantRecordHandler {
+    private static final Logger logger = LoggerFactory.getLogger(ErrantRecordHandler.class);
+    private final ErrantRecordReporter errantRecordReporter;
+
+    private static final List<String> allowedBigQueryErrorReason = Arrays.asList("invalid");
```

Comment:
Using a set would be more efficient in lookup?

Reply:
I want to do a case insensitive match where I will need to traverse entire list/set. So a set won't bring in much difference as we are traversing all the values and doing a case insensitive match. 

Also if we ignore this case insensitive match thing and just lookup error reason by set lookup method, there won't be much difference in performance here as `allowedBigQueryReason` list won't be very long.